Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '>= 2.3.8', '< 2.4' # must be < 2.4 until we upgrade to Rails >= 4.2.8 (see https://rubyonrails.org/2017/2/21/Rails-4-2-8-has-been-released)
+ruby '>= 2.4.10', '< 2.5'
```

Comment:
This is dropping support for 2.3.8, just checking that this is on purpose?

Reply:
yes, in that it limits the user to the same versions that are actually running on the server